tests/robustness/coverage/README.md

```diff
@@ -0,0 +1,60 @@
+## Overview
+
+Go tests in this directory analyze the usage of Etcd API based on collected
+traces from a Kubernetes cluster. They output information on:
+
+1. Number of calls per gRPC method used by Kubernetes
+
+This information can be used to track the coverage of k8s-etcd contract.
+
+### Manual test execution
+
+At first we will manually set up the cluster, run e2e tests, download traces and
+then execute the test.
+
+1\. Set up [KIND
+cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) with
+tracing exporting to [Jaeger](https://www.jaegertracing.io/)
+
+```
+export KUBECONFIG="$(pwd)/kind-with-tracing-config"
```

Comment:
Not sure I like you kubeconfig manipulation. Is it just to avoid overriding previously existing kubeconfig? Doesn't seem important.  I would leave kubeconfig management to the user. We could comment that running this procedure will override kubeconfig, but I would skip `export KUBECONFIG` and `kind export kubeconfig`.

Reply:
You mean that it will fail with just `kind export kubeconfig` ? meaning it will not default to `~/.
kube/config`?

Reply:
if `KUBECONFIG` environment variable is not set I get the following error from k8s e2e tests:
```
  I0707 12:29:24.864334 3247356 util.go:453] >>> kubeConfig:
  I0707 12:29:24.864384 3247356 e2e.go:186] Unexpected error: Error loading client:
      <*errors.errorString | 0xc00038c460>:
      error creating client: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
      {
          s: "error creating client: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined",
      }
  [FAILED] in [SynchronizedBeforeSuite] - k8s.io/kubernetes/test/e2e/e2e.go:186 @ 07/07/25 12:29:24.864
```

Reply:
Ok